### PR TITLE
Release: AEMRules v1.0

### PR DESCRIPTION
**Pull request by jplucinski**
### AEMRules v1.0

Release notes:
- HTL Support
- Dropped Slice Support

### CODE REVIEW

File: aemrules.properties [hunk 1]
```diff
@@ -29,4 +35,4 @@ defaults.mavenArtifactId=aemrules
 0.8.sqVersions=[5.6.1,6.3]
 0.8.date=2015-08-31
 0.8.changelogUrl=https://github.com/Cognifide/AEM-Rules-for-SonarQube/releases/tag/v0.8
-0.8.downloadUrl=https://github.com/Cognifide/AEM-Rules-for-SonarQube/releases/download/v0.8/aemrules-0.8.jar
+0.8.downloadUrl=https://github.com/Cognifide/AEM-Rules-for-SonarQube/releases/download/v0.8/aemrules-0.8.jar
```
Comment (ganncamp) [inline aemrules.properties]: GH wants you to reinstate the final linebreak here.

File: aemrules.properties [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
+defaults.mavenArtifactId=sonar-aemrules-plugin
+
+1.0.description=HTL Support
+1.0.sqVersions=[6.7,LATEST]
+1.0.date=2020-02-07
+1.0.changelogUrl=https://github.com/Cognifide/AEM-Rules-for-SonarQube/releases/tag/v1.0
+1.0.downloadUrl=https://github.com/Cognifide/AEM-Rules-for-SonarQube/releases/download/v1.0/sonar-aemrules-plugin-1.0.jar
 
 0.11.description=Fix use of removed dependency, add two rules
 0.11.sqVersions=[6.7,LATEST]
```
Comment (ganncamp) [inline aemrules.properties]: Only one version can be compatible with `LATEST` at a time. Best practice is to set the range end of the old version to the current SQ version.

File: aemrules.properties [hunk 3]
```diff
@@ -1,11 +1,17 @@
...
-publicVersions=0.11
+archivedVersions=0.8,0.9,0.10,0.11
+publicVersions=1.0
 
 defaults.mavenGroupId=com.cognifide.aemrules
-defaults.mavenArtifactId=aemrules
+defaults.mavenArtifactId=sonar-aemrules-plugin
+
+1.0.description=HTL Support
+1.0.sqVersions=[6.7,LATEST]
```
Comment (ganncamp) [inline aemrules.properties]: SQ versions <7.9 have been archived. Please set the beginning of this range to 7.9.

File: aemrules.properties (hunk 1 above)
Comment (jplucinski) [reply aemrules.properties]: Fixed.

File: aemrules.properties (hunk 2 above)
Comment (jplucinski) [reply aemrules.properties]: Fixed.

File: aemrules.properties (hunk 3 above)
Comment (jplucinski) [reply aemrules.properties]: Fixed.

**Pull request merged**